systems/frames/MultiCoordinateFrame.m

```diff
@@ -293,10 +293,54 @@ function display(obj)
       fr = obj.frame{id};
     end
     
+    function fr = getFrameByNameRecursive(obj,name)
```

Comment:
i like these, but you're not actually using them anywhere... (unless I missed it?).  If we want them in, we should have at least a unit test so the code doesn't go stale..?